core/store/models/job_spec_v2.go

```diff
@@ -13,39 +14,56 @@ import (
-		IsBootstrapPeer                        bool           `toml:"isBootstrapPeer"`
-		EncryptedOCRKeyBundleID                Sha256Hash     `toml:"keyBundleID"                 gorm:"type:bytea"`
-		MonitoringEndpoint                     string         `toml:"monitoringEndpoint"`
-		TransmitterAddress                     EIP55Address   `toml:"transmitterAddress"`
-		ObservationTimeout                     Interval       `toml:"observationTimeout" gorm:"type:bigint"`
-		BlockchainTimeout                      Interval       `toml:"blockchainTimeout" gorm:"type:bigint"`
-		ContractConfigTrackerSubscribeInterval Interval       `toml:"contractConfigTrackerSubscribeInterval"`
-		ContractConfigTrackerPollInterval      Interval       `toml:"contractConfigTrackerPollInterval" gorm:"type:bigint"`
-		ContractConfigConfirmations            uint16         `toml:"contractConfigConfirmations"`
-		CreatedAt                              time.Time      `toml:"-"`
-		UpdatedAt                              time.Time      `toml:"-"`
+		ID                                     int32          `json:"-" toml:"-"                 gorm:"primary_key"`
+		ContractAddress                        EIP55Address   `json:"contractAddress" toml:"contractAddress"`
+		P2PPeerID                              PeerID         `json:"p2pPeerID" toml:"p2pPeerID"         gorm:"column:p2p_peer_id"`
+		P2PBootstrapPeers                      pq.StringArray `json:"p2pBootstrapPeers" toml:"p2pBootstrapPeers" gorm:"column:p2p_bootstrap_peers;type:text[]"`
+		IsBootstrapPeer                        bool           `json:"isBootstrapPeer" toml:"isBootstrapPeer"`
+		EncryptedOCRKeyBundleID                Sha256Hash     `json:"keyBundleID" toml:"keyBundleID"                 gorm:"type:bytea"`
+		MonitoringEndpoint                     string         `json:"monitoringEndpoint" toml:"monitoringEndpoint"`
+		TransmitterAddress                     EIP55Address   `json:"transmitterAddress" toml:"transmitterAddress"`
+		ObservationTimeout                     Interval       `json:"observationTimeout" toml:"observationTimeout" gorm:"type:bigint"`
+		BlockchainTimeout                      Interval       `json:"blockchainTimeout" toml:"blockchainTimeout" gorm:"type:bigint"`
+		ContractConfigTrackerSubscribeInterval Interval       `json:"contractConfigTrackerSubscribeInterval" toml:"contractConfigTrackerSubscribeInterval"`
+		ContractConfigTrackerPollInterval      Interval       `json:"contractConfigTrackerPollInterval" toml:"contractConfigTrackerPollInterval" gorm:"type:bigint"`
+		ContractConfigConfirmations            uint16         `json:"contractConfigConfirmations" toml:"contractConfigConfirmations"`
+		CreatedAt                              time.Time      `json:"createdAt" toml:"-"`
+		UpdatedAt                              time.Time      `json:"updatedAt" toml:"-"`
 	}
 
 	PeerID peer.ID
 )
 
+func (js JobSpecV2) GetID() string {
```

Comment:
Are these IDs used for element identification in typescript-land?

If so, you may want to prepend a string in front otherwise there could be overlap (these are not UUIDs).